_docs/admin/setup/mail/mail-routing.md

```diff
@@ -15,8 +15,28 @@ Navigate to **Setup >> Mail >> Incoming Mail >> Mail Routing**.
 
 You can configure rules to automatically route new mail to groups based on message properties.
 
-For example, you may want to route messages addressed to `support@*` to the **Support** group, `orders@*` to **Sales**, and `receipts@*` to **Billing**.
+For example, you may want to route messages addressed to `support@*` to the **Support** group, `orders@*` to **Sales**, and `receipts@*` to **Billing**. Adding an <b>*</b> after the **@** symbol makes the rule a wildcard that link multiple domains to that mailbox. 
```

Comment:
Use Markdown syntax here, not HTML. You could highlight it with code format instead:

~~~
"Adding an asterisk (`*`) ...
~~~